zkevm/src/circuit.rs

```diff
@@ -32,11 +32,11 @@ pub trait TargetCircuit {
     /// used to generate vk&pk
     fn empty() -> Self::Inner;
     //fn public_input_len() -> usize { 0 }
-    fn from_block_result(block_result: &BlockResult) -> anyhow::Result<(Self::Inner, Vec<Vec<Fr>>)>
+    fn from_block_result(block_result: &BlockTrace) -> anyhow::Result<(Self::Inner, Vec<Vec<Fr>>)>
```

Comment:
rename block_result(s) to block_trace(s) in 
+ zkevm/src/circuit.rs
+ zkevm/src/circuit/builder.rs
+ zkevm/src/prover.rs
+ zkevm/src/utils.rs
+ zkevm/tests/integration.rs